src/m2n/PointToPointCommunication.cpp

```diff
@@ -773,6 +773,34 @@ void PointToPointCommunication::broadcastReceive(double &itemToReceive)
   }  
 }
 
+void PointToPointCommunication::broadcastSendMesh(mesh::Mesh &mesh)
```

Comment:
The mesh is already a member of this class. No need to pass it as an argument again, right?

Reply:
I am not sure if it's the same one. we give this mesh from partion mackage to m2n.cpp and from there we give it to methods in  the PointToPointCommunication.cpp class.

Reply:
It should be the same mesh, no need to duplicate that.